### ISSUE TITLE
Visual subdivision or separation of each post in the Card view

### ISSUE THREAD

**Issue by araccaine** [LemmyNet/jerboa#1568]
### Pre-Flight checklist

- [X] Did you check to see if this issue already exists?
- [X] This is a single feature request. (Do not put multiple feature requests in one issue)
- [X] This is not a question or discussion. (Use https://lemmy.ml/c/jerboa for that)

## Current behaviour

When browsing a community, the posts are not really divided from each other:

![Screenshot_20240625-094350_Jerboa_1](https://github.com/LemmyNet/jerboa/assets/75312500/b49666b2-9a6a-4469-88e8-0d75fbb2ebe9)

## Feature request

Maybe a horizontal line can be added, or some card highlighting? Just to be able to distinguish each post from another:

![image7](https://github.com/LemmyNet/jerboa/assets/75312500/0f85f398-6cf0-46b1-9075-0e46acb6b25e)

![image8](https://github.com/LemmyNet/jerboa/assets/75312500/1f00fe2d-95cc-40a2-b9fc-29e997098b62)

(Did the pictures on my phone, no hate plz 🌝 Just to get an idea)

## Notes

I used these settings for the screenshots, but also tested other settings:

- Theme: System
- Theme Color: Dracula
- Post View: Card

**Comment by dessalines** [comment]
Yep that is true. I removed the horizontal dividers after looking at several other reddit and lemmy apps, and realizing that they weren't necessary. 

It doesn't matter to me too much either way, as long as we keep a mostly flat design.

**Comment by MV-GH** [comment]
These are back so i am closing this
